internal/uvm/create_lcow.go

```diff
@@ -156,6 +155,12 @@ func NewDefaultOptionsLCOW(id, owner string) *OptionsLCOW {
 		DisableTimeSyncService:  false,
 	}
 
+	// if VPMEM Multimapping is disabled we should enable 4 SCSI controllers so that we can still
+	// support upto 128 container image layers.
+	if osversion.Build() >= osversion.RS5 && opts.VPMemNoMultiMapping {
```

Comment:
Can we expect `VPMemNoMultiMapping` to be `true` when the VPMEM count is 0? Or should we also add "|| vpmem count == 0" to this conditional

Reply:
Changed this to only check if os version is > RS5. Now it is the user's responsibility to provide the correct annotation to set VPMem device count to 0 (and rootfs type of initrd).